static/usr/lib/systemd/system/tee-supplicant.service.d/core-override.conf

```diff
@@ -0,0 +1,12 @@
+# SPDX-License-Identifier: BSD-2-Clause
+# Adapted from https://github.com/OP-TEE/optee_client/blob/6486773583b5983af8250a47cf07eca938e0e422/tee-supplicant/tee-supplicant%40.service.in
+[Unit]
+DefaultDependencies=no
+BindsTo=dev-teepriv0.device var-lib-optee\x2dclient-data-tee.mount
+After=dev-teepriv0.device var-lib-optee\x2dclient-data-tee.mount
+
+[Service]
+Type=notify
```

Comment:
I am not sure you should override this one. This is a drop-in. And the original looks like having that set.

Reply:
Eventually, we should maybe build it ourselves. That is a universe package anyway.